tests/yadism/cf/test_coupling_constants.py

```diff
@@ -38,11 +43,11 @@ def test_get_weight(self):
             sin2theta_weak=0.5,
             CKM="0.97428 0.22530 0.003470 0.22520 0.97345 0.041000 0.00862 0.04030 0.999152",
         )
-        obs_d = dict(prDIS="CC", PolarizationDIS=0.0, PropagatorCorrection=0)
-        assert coupl.CouplingConstants.from_dict(th_d, obs_d).get_weight(
-            1, 0, None, cc_mask="b"
-        ) == coupl.CouplingConstants.from_dict(th_d, obs_d).get_weight(
-            1, 0, None, cc_mask="b"
+        obs_d = dict(
+            prDIS="CC",
+            PolarizationDIS=0.0,
+            PropagatorCorrection=0,
+            NCPositivityCharge=None,
         )
```

Comment:
Do your remember why we were doing this apparently trivial test?

Reply:
no idea ... https://github.com/N3PDF/yadism/pull/146/commits/dff8b30b33fdbdce77aa345877538709a7fc866d makes it more non-trivial

Reply:
Maybe it was only too much copy-paste
(I was considering if we were checking some kind of memory effect - too involved, Occam's razor suggest a poor implementation)

Thank you for noticing and the improvement